### DISCUSSION TITLE
[Kavita] Enhancement ideas/requests to Browse Authors

### DISCUSSION THREAD

**Discussion by jncmney** [Kareadita/Kavita#3416] [Ideas]
### Idea Description

The new Browse Authors section is brilliant and I'm finding I use it quite a lot more than I thought.

A few ideas/feature requests:

- Favourite/Star Authors - The ability to mark temporarily some Authors as Favourites, and a section in the UI to see those, would be great. A use case is if I've read several works by an author and I want to read everything they've done, this is a great way to track progress and see what else the author has worked on that I haven't read.
- Sort by number of issues or series - I'd like to be able to sort the Authors by "volume of work I have" to see which ones I read the most.

### Idea Category

Feature Enhancement

### Duration of Using Kavita

A few years

### Before submitting

- [X] I've already searched for existing ideas before posting.

**Comment by majora2007** [comment]
I like these ideas, I'd like to leave this open and see if we can gather any more ideas.
